hub/management/commands/import_council_carbon_neutral_data.py

```diff
@@ -0,0 +1,94 @@
+import pandas as pd
+from mysoc_dataset import get_dataset_url
+
+from hub.import_utils import filter_authority_type
+from hub.models import DataSet
+
+from .base_importers import BaseImportFromDataFrameCommand, MultipleAreaTypesMixin
+
+declare_map = {
+    "Y": "Yes",
+    "N": "No",
+}
+
+
+class Command(MultipleAreaTypesMixin, BaseImportFromDataFrameCommand):
+    cons_row = "gss_code"
+    message = "Importing council net zero target data"
+    uses_gss = True
+    do_not_convert = True
+
+    area_types = ["STC", "DIS"]
+
+    defaults = {
+        "label": "Net Zero Date",
+        "data_type": "integer",
+        "category": "place",
+        "subcategory": "date",
+        "source_label": "Data from mySociety",
+        "source": "https://pages.mysociety.org/la-plans-promises/",
+        "source_type": "csv",
+        "table": "areadata",
+        "data_url": "https://pages.mysociety.org/la-plans-promises/downloads/local-authority-climate-emergency-declarations-declarations-csv/latest",
+        "comparators": DataSet.numerical_comparators(),
+        "is_filterable": True,
+        "is_shadable": True,
+        "is_public": True,
+        "unit_type": "raw",
+        "unit_distribution": "physical_area",
+        "fill_blanks": False,
+    }
+
+    data_sets = {
+        "council_net_zero_date": {
+            "defaults": defaults,
+            "col": "year",
+        },
+        "council_net_zero_details": {
+            "defaults": {
+                **defaults,
+                "data_type": "json",
+                "label": "Net Zero Details",
```

Comment:
Guess the wording of this dataset doesn’t matter, as it’s only for internal use. But we tend to favour sentence-case for dataset names (rather than capitalising every single word) – so this would be “Net Zero details”, or I guess, “Net Zero target date details”.